### PR TITLE
Initial add of SPI section to Summit docs

### PR DESCRIPTION
**Pull request by chrisfuson**

### CODE REVIEW

File: systems/summit_user_guide.rst [hunk 1]
```diff
@@ -4293,6 +4293,97 @@ judiciously, especially when running the debugger in "batch" or
...
+========================
+
+.. _enhanced-enclave:
+
+Enhanced Enclave
+------------
+The OLCF provides multiple security enclaves to support multiple levels of computing. 
+
+* Open
+  * Training resources such as Andes is an example
```
Comment (LeahHuk) [inline systems/summit_user_guide.rst:4309]: Did you mean Ascent?

**Review by LeahHuk — CHANGES_REQUESTED** [review]
I assume its meant to be bulleted skeleton for now as a jumping off point. I only saw the one issue with the Open enclave section where I think you meant to say Ascent and not Andes. I can work on turning the bullet points into actual text.

File: systems/summit_user_guide.rst (hunk 1 above)
Comment (chrisfuson) [reply systems/summit_user_guide.rst]: Yes; fixed

**Comment by chrisfuson** [comment]
> 

I can format differently as the next step if bulleted is not the best in this case.

**Review by LeahHuk — APPROVED** [review]

**Pull request merged**